crates/stackable-operator/src/commons/opa.rs

```diff
@@ -44,14 +44,33 @@
 //! assert_eq!(opa_config.full_document_url(&cluster, "http://localhost:8081", None, OpaApiVersion::V1), "http://localhost:8081/v1/data/test".to_string());
 //! ```
 use crate::client::{Client, GetApi};
-use crate::error;
-use crate::error::OperatorResult;
 use k8s_openapi::{api::core::v1::ConfigMap, NamespaceResourceScope};
 use kube::{Resource, ResourceExt};
 use lazy_static::lazy_static;
 use regex::Regex;
 use schemars::{self, JsonSchema};
 use serde::{Deserialize, Serialize};
+use snafu::{OptionExt, ResultExt, Snafu};
+
+type Result<T, E = Error> = std::result::Result<T, E>;
+
+#[derive(Debug, Snafu)]
+pub enum Error {
+    #[snafu(display("cannot find OPA configmap {configmap_name:?} in namespace {namespace:?}"))]
+    GetOpaConfigMap {
+        source: crate::client::Error,
+        configmap_name: String,
```

Comment:
I'm not quite sure how I can with this one, since we only have access to a `ConfigMap` name, and not the actual resource (`&self` is an `&OpaConfig`).